distrib/starlake.cmd

```diff
@@ -74,7 +107,21 @@ if "%javaVersion%" LSS "11000" (
 )
 
 set setup_url=https://raw.githubusercontent.com/starlake-ai/starlake/master/distrib/setup.jar
-curl -s -o %SCRIPT_DIR%setup.jar %setup_url%
+if defined https_proxy (set "proxy=%https_proxy%") else if defined http_proxy (set "proxy=%http_proxy%") else (echo no proxy)
+if not "%proxy%"=="" (
```

Comment:
The script does not work if openssl is not installed. This makes openssl a requirement

Reply:
it is required only if a proxy has been found...

The script simply doesn't work otherwise

Reply:
I had no issue when behind a proxy from my office with the current script

Reply:
Well maybe this PR should be closed :) except for this 2 fixes : 

- when using a proxy which does not require any authentication (Setup.java line 55 to 61)
- when installed using starlake.sh from Windows (using git bash)